### ISSUE TITLE
where is the OSMBuildings.js file?

### ISSUE THREAD

**Issue by ZhenanH** [OSMBuildings/OSMBuildings#28]
by the way, does it work with Mabox-gl-js now? thanks

**Comment by rbuch703** [comment]
TL;DR: to get the `OSMBuildings.js` file download a release (for example https://github.com/OSMBuildings/OSMBuildings/releases/download/v2.2.1/OSMBuildings-2.2.1.zip).

Long version: We no longer store the compiled output files inside the OSMBuildings GitHub repository. OSMBuildings consists of about 50 JavaScript files and about 20 GLSL shader. It is subdivided into this set of files in order to simplify development of OSMBuildings. For *users* of OSMBuildings, however, this subdivision  is not desirable. It would mean that you'd have to include all 70 source files in your web page, and they would be loaded separately, increasing the load times of your web site.

So instead, we automatically compile these 70 source files (JavaScript and GLSL) into a single `OSMBuildings.js` file using the build system *grunt*. Before, we updated this `OSMBuildings.js` inside the repository pretty much whenever we changed anything in the source files. But this was hard to do (we sometimes forgot, leading to an `OSMBuildings.js` that had different bugs/features than the actual source files), and it cluttered the git repository (which contains every single version of this file ever committed and thus unnecessarily grows in size).

So we decided to no longer have the compiled output file inside the repository. The OSMBuildings repository now only contains the actual source files. If you want a well-tested and stable version of OSMBuildings, don't clone the repository, but simply click on the "releases" link on the repository's main page and download the most recent release of OSMBuildings from that page. If, instead, you want bleeding-edge features of OSMBuildings and don't mind occasional bugs, you can use *grunt* to create the `OSMBuildings.js` yourself. The details vary based on your platform, but the general steps to take are these:
- install nodejs and npm (if not contained in your nodejs package)
- clone the OSMBuildings repository
- open a terminal/shell inside the cloned folder
- execute `sudo npm -g grunt-cli`
- execute `sudo npm install`
- execute `grunt release`

This should create the OSMBuildings.js file in the `dist` folder.

**Comment by kekscom** [comment]
@rbuch703 Thank you for the good explanation!
I should put that to a docs section.

@ZhenanH It's not working with MapboxGL yet.

**Comment by ZhenanH** [comment]
@rbuch703  Thanks a lot!!
